packages/experimental/package.json

```diff
@@ -25,12 +25,12 @@
         "clean": "rimraf dist tsconfig.build.tsbuildinfo"
     },
     "peerDependencies": {
-        "@sharegate/orbit-ui": ">=34.1.2",
+        "@sharegate/orbit-ui": ">=34.2.0",
```

Comment:
this seems to cause major versionning everytime a new version is released. I'll investigate